include/mbgl/gl/location_component_layer.hpp

```diff
@@ -0,0 +1,114 @@
+#pragma once
```

Comment:
This file shall be generated (I guess it was discussed before)

Reply:
do you suggest to remove the convenience function i added (setLocation(const LanLng& ), so that it can be used as straight from the generator?

Reply:
Could you add an aux conversion function `LatLng` -> `PropertyValue<std::array<double, 3> >`? or alternatively can generate a base class and inherit from it adding hand-written extensions.

Reply:
The second option seems like a stretch, the generated Layer classes contain a number of final methods that would have to be removed, at which point the goal of keeping generated files as-is would vanish.
The first aux function could work, but note that the generated file atm have to be at least minimally modified as they are in mbgl/gl, so inclusions have to be corrected

Reply:
done in the first way in the end